app/controllers/admin/budget_groups_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   end
 
   def new
-    @group = @budget.groups.new
+    @group = @budget.groups.new(voting_cant_exceed_heading_budget: false)
```

Comment:
Naming is always tricky, we try to keep names short, descriptive and positive 😌I wonder if there is a better name for this 🤔What would be a good word to describe being limited by money? 🤔 Maybe... `bounded`, `restricted` (as you use in specs), `money_bound`. Suggestions welcomed 😌

Reply:
I was thinking to use `restricted` but was worried that some other feature might find the name more appropriate. I am happy to any. I will use bounded since its the first thing in the list?

Reply:
I am going to use money_bound. It hints what it is.